CMakeLists.txt

```diff
@@ -77,9 +77,9 @@ include(GetGitRevisionDescription)
 get_git_head_revision(GIT_REFSPEC GIT_SHA1)
 
 add_definitions(
+    -DQT_DISABLE_DEPRECATED_BEFORE=0
```

Comment:
I disagree and according to [QT_DISABLE_DEPRECATED_BEFORE Documentation](https://doc.qt.io/qt-5/qtglobal.html#QT_DISABLE_DEPRECATED_BEFORE) I would have expected something like `QT_DISABLE_DEPRECATED_BEFORE=0x051500`

Reply:
Why do you disagree? If we don't disable the deprecation warnings we can't enable warning as errors. A few deprecation warnings are not solvable at the moment with Qt5. I don't think it helps us in day to day work to see these warnings.

Reply:
is it not possible to for example enable those from Qt 5.14 ?
I guess we really only have a problem with deprecations from QtNetwork that are only solvable with Qt 6
given I may miss something, please correct me if needed

Reply:
I think there are some older ones too. That flag was enabled already for src/gui.

Reply:
and the desktop client does not compile without it.

Reply:
OK something for next Friday !